support/Pipelines/Bbh/EccentricityControl.py

```diff
@@ -35,34 +35,23 @@ def eccentricity_control(
 
     This function does the following:
 
-    - Reads orbital parameters from the Inspiral.yaml input file. These
-      quantities are stored in the h5_file and are angular velocity and
-      expansion.
+    - Reads orbital parameters from the 'id_input_file_path'.
 
     - For now, the time boundaries are set to start at 500, and at the end of
       the simulation to use all available data, but we will make tmin and tmax
       more dynamic later, so the user can change those variables according to
       interest.
 
     Arguments:
-      h5_file: file that contains the orbital parameters; required parameter
-        when calling the function
-      subfile_name_aha: subfile for black hole A; required parameter when
-        calling the function
-      subfile_name_ahb: subfile for black hole B; required parameter when
-        calling the function
-      angular_velocity_from_xcts: orbital parameter we are interested in
-        updating; optional parameter, if none is provided, defaults to None
-      expansion_from_xcts: orbital parameter we are interested in updating;
-        optional parameter, if none is provided, defaults to None
-      tmin: sets starting point for eccentricity reduction script; defaults to
+      h5_file: file that contains the trajectory data; optional parameter
+      id_input_file_path: path to the input file of the initial data run
+      subfile_name_aha: subfile for black hole A; optional parameter
+      subfile_name_ahb: subfile for black hole B; optional parameter
+      tmin: starting point for eccentricity reduction script; defaults to
         500 if not specified
       tmax: stopping point for eccentricity reduction script; set to use all
         available data by default
-      output: represents output of the function; optional parameter, if none
-        is provided, defaults to None
-      id_input_file_path: path to the input file of the initial data run; reads
-        from Inspiral.yaml, if none provided, defaults to None.
+      output: represents output of the function; optional parameter
```

Comment:
Please be more specific. Should this be a directory where plots will be written to, or a file path for a plot? I don't remember.